File: easy/q9/q9.java
package easy.q9;
class q9 {
    public boolean isPalindrome(int x) {
        String s = String.valueOf(x);
        char[] array = s.toCharArray();
        for (int i = 0; i < array.length / 2; i++) {
            if (array[i] != array[array.length - 1 - i]) {
                return false;
            }
        }
        return true;
    }
}

File: easy/q9/q9.py
class Solution:
    def isPalindrome(self, x: int) -> bool:
        s = str(x)
        array = list(s)

        for i in range(len(array) // 2):
            if array[i] != array[len(array) - 1 - i]:
                return False

        return True